docs/quick-start.md

```diff
@@ -119,7 +119,7 @@ The source code of the `even-odd` [User Defined Function](user-guide/user-define
 The pipeline can be deleted by
 
 ```shell
-kubectl delete -f https://raw.githubusercontent.com/numaproj/numaflow/stable/test/e2e/testdata/even-odd.yaml
+kubectl delete -f https://raw.githubusercontent.com/numaproj/numaflow/stable/examples/2-even-odd-pipeline.yaml
```

Comment:
Use `main` instead of `stable` in the URL for now - the link won't be work after merging this.

Reply:
Oh I don't know about this. How come the `kubectl apply -f https://raw.githubusercontent.com/numaproj/numaflow/stable/examples/1-simple-pipeline.yaml
` above works?

Reply:
that's right.